cvat-ui/src/components/actions-menu/actions-menu.tsx

```diff
@@ -21,7 +21,7 @@ interface Props {
     dumpActivities: string[] | null;
     exportActivities: string[] | null;
     inferenceIsActive: boolean;
-
+    taskDimension: string;
```

Comment:
Same here (enum)

Reply:
Since there are no redux actions performed on the taskDimension, we have used string here.
If we need to change to enum is there any common location where we can place and use it in whole of cvat-ui, kindly suggest.

Reply:
For now you can place it to "**/cvat-ui/src/reducers/interfaces.ts**" like this, for example:
```
export enum TaskDimension {
    D2 = '2D',
    D3 = '3D',
}
```

right after, for example:
```
export enum TaskStatus {
    ANNOTATION = 'annotation',
    REVIEW = 'validation',
    COMPLETED = 'completed',
}
```

I would propose try to not use a plain strings, especially if it's some kind of constant options or some stuff like this

Reply:
Changed code accordingly.